txmongo/collection.py

```diff
@@ -91,7 +94,9 @@ def __init__(self, database, name, write_concern=None, codec_options=None):
             msg = "TxMongo: collection names must not contain '$', '{0}'".format(repr(name))
             raise InvalidName(msg)
         if name[0] == "." or name[-1] == ".":
-            msg = "TxMongo: collection names must not start or end with '.', '{0}'".format(repr(name))
+            msg = "TxMongo: collection names must not start or end with '.', '{0}'".format(
+                repr(name)
+            )
```

Comment:
Shouldn't this fit on one line with 120?

Reply:
Yes, I think I might have gotten over aggressive on those. I will fix them.